lib/roby/interface/client.rb

```diff
@@ -324,6 +324,7 @@ def call(path, m, *args)
             # @param [Symbol] m command or action name. Actions are always
             #   formatted as action_name!
             # @param [Object] args the command or action arguments
+            # @return [Hash] the hash describing the async call
```

Comment:
Don't let people expect a hash here. Just return "an Object that can be used to test for the call presence with `#async_call_pending?`" (even if you do return the hash)

Reply:
Any particular reason? The information in the hash may actually be useful for the user to track what is pending